src/css/main.less

```diff
@@ -61,6 +62,7 @@ a:focus {
 a,
 .btn {
   transition: all 0.2s ease-in-out;
+  padding: 7px 15px;
```

Comment:
Yes the naming is weird.
Consider to introduce a generic var with 14px in it and use that.